Code/GraphMol/testSGroup.cpp

```diff
@@ -1018,13 +1021,180 @@ void testSubstanceGroupsAndRemoveHs(const std::string &rdbase) {
       ps.removeInSGroups = true;
       MolOps::removeHs(mol_copy, ps);
       TEST_ASSERT(mol_copy.getNumAtoms() == 6);
-      TEST_ASSERT(getSubstanceGroups(mol_copy).size() == 0);
+      TEST_ASSERT(getSubstanceGroups(mol_copy).size() == 1);
     }
     {  // check removeAllHs() too
       RWMol mol_copy = *mol;
       MolOps::removeAllHs(mol_copy);
       TEST_ASSERT(mol_copy.getNumAtoms() == 6);
-      TEST_ASSERT(getSubstanceGroups(mol_copy).size() == 0);
+      TEST_ASSERT(getSubstanceGroups(mol_copy).size() == 1);
+    }
+  }
+}
+
+void testKeepSpecialHsOnRemoval() {
+  BOOST_LOG(rdInfoLog) << "-----------------------\n Testing github XXXX: "
```

Comment:
```suggestion
  BOOST_LOG(rdInfoLog) << "-----------------------\n Testing github 5399: "
```

Reply:
Ha, I figured I'd update that later on, but then forgot.